src/App.tsx

```diff
@@ -3,10 +3,12 @@ import { BackendConnectionChecker } from "./components/BackendConnectionChecker"
 import { Home } from "./components/Home/Home";
 import React from "react";
 import { Route, Routes } from "react-router-dom";
+import { Navbar } from './components/Navigation/NavBar';
 
 function App() {
   return (
     <>
+    <header><Navbar/></header>
```

Comment:
It is better practice to have the `<header>` tags inside the Navbar component although I think in this case it is not necessary as we are using the `<nav>` tag